src/main/java/dev/tycho/stonks/managers/DatabaseHelper.java

```diff
@@ -857,6 +858,246 @@ public void changeVerification(Player player, String companyName, boolean newVer
+    } catch (SQLException e) {
+      e.printStackTrace();
+      player.sendMessage(ChatColor.RED + "SQL EXCEPTION TELL WHEEZY");
+    }
+  }
+
+  public void subscribeToService(Player player, int serviceId, Boolean autoPay) {
+    Service service;
+    try {
+      service = databaseManager.getServiceDao().queryForId(serviceId);
+    } catch (SQLException e) {
+      e.printStackTrace();
+      player.sendMessage(ChatColor.RED + "SQL EXCEPTION TELL WHEEZY");
+      return;
+    }
+    if (service == null) {
+      player.sendMessage(ChatColor.RED + "That service id does not exist");
+      return;
+    }
+    if (service.getMaxSubscriptions() > 0 && service.getNumSubscriptions() >= service.getMaxSubscriptions()) {
+      player.sendMessage(ChatColor.RED + "That service has the maximum number of subscriptions");
+      return;
+    }
+
+    if (service.getSubscription(player) != null) {
+      player.sendMessage(ChatColor.RED + "You cannot subscribe to the same service twice");
+      return;
+    }
+
+    new ConfirmationGui.Builder().title("Accept first bill of $" + service.getCost()).onChoiceMade(
+        c -> {
+          if (!c) return;
+          //We can now subscribe to the service
+          //Setup the subscription
+          Subscription subscription = new Subscription(player, service, autoPay);
+          if (!Stonks.economy.withdrawPlayer(player, service.getCost()).transactionSuccess()) {
+            player.sendMessage(ChatColor.RED + "Insufficient funds!");
+            return;
+          } else {
+            //Payment success
+            try {
+              databaseManager.getSubscriptionDao().create(subscription);
+              service.getAccount().getAccount().addBalance(service.getCost());
+              databaseManager.logTransaction(new Transaction(service.getAccount(),
+                  player.getUniqueId(), "Subscription payment for " + service.getName(), service.getCost()));
+              //Update the account database
+              databaseManager.updateAccount(service.getAccount().getAccount());
+
+
+              //Subscription created!
+              player.sendMessage(ChatColor.GREEN + "You have subscribed to the service " + service.getName() +
+                  " (provided by " + service.getCompany().getName() + ")");
+              player.sendMessage(ChatColor.GREEN + "This service will expire in " + service.getDuration() + " days");
+              if (autoPay) {
+                player.sendMessage(ChatColor.GREEN + "You have set your subscription to automatically renew, so you don't need to do anything.");
+              } else {
+                player.sendMessage(ChatColor.GREEN + "You have set your subscription to manually renew, so you will need to resubscribe in "
+                    + service.getDuration() + " days time.");
+              }
+            } catch (SQLException e) {
+              e.printStackTrace();
+              player.sendMessage(ChatColor.RED + "SQL EXCEPTION TELL WHEEZY");
```

Comment:
Use `sendMessage(Player, String)` and don't use `ChatColor.RED` (Also follow the messages i've used for errors)